### PR TITLE
ELE-673 slack summary remove attention required section

### PR DESCRIPTION
**Pull request by IDoneShaveIt**
Removed the attention required section from the report summary message.
Reformatted the alert structure to make it look good without it.
image
image

### CODE REVIEW

File: tests/unit/monitor/data_monitoring/report/test_slack_report_summary_message_builder.py [hunk 1]
```diff
@@ -52,35 +52,12 @@ def test_add_details_to_slack_alert_attachments_limit(test_results_summary):
     assert "The amount of results exceeded Slack" in attachments_as_string
 
 
-def test_owners_tags_and_subscribers_of_passed_tests_are_filtered_out(
```
Comment (IDoneShaveIt) [inline tests/unit/monitor/data_monitoring/report/test_slack_report_summary_message_builder.py:55]: This test is no longer relevant because we removed the tags / owners / subscribers from the summary

File: tests/unit/monitor/data_monitoring/report/test_slack_report_summary_message_builder.py [hunk 2]
```diff
@@ -52,35 +52,12 @@ def test_add_details_to_slack_alert_attachments_limit(test_results_summary):
...
-    assert "staging" not in attachments_as_string
-
-
 def test_passed_tests_filtered_out_of_details_view(
     test_results_summary,
 ):
     # Within attachments limitation
     message_builder = SlackReportSummaryMessageBuilder()
     passed_tests_from_fixture = [x for x in test_results_summary if x.status == "pass"]
-    message_builder._add_preview_to_slack_alert(passed_tests_from_fixture)
```
Comment (IDoneShaveIt) [inline tests/unit/monitor/data_monitoring/report/test_slack_report_summary_message_builder.py:83]: Not needed for the test

File: docs/guides/share-observability-report/send-report-summary.mdx [hunk 3]
```diff
@@ -11,51 +11,38 @@ This is useful for you and your team members to understand quickly if there is s
...
 To filter by tag:
+
 ```shell send-report selectors
 edr send-report --select tag:finance
 edr send-report --select config.meta.owner:@jeff
 edr send-report --select model:customers
 edr send-report --select customers
 ```
-
-#### Include test descriptions
```
Comment (RoiTabach) [inline docs/guides/share-observability-report/send-report-summary.mdx:52]: Why is this removed?

**Review by RoiTabach — APPROVED** [review]
Small docs question, otherwise go for merge

File: docs/guides/share-observability-report/send-report-summary.mdx (hunk 3 above)
Comment (IDoneShaveIt) [reply docs/guides/share-observability-report/send-report-summary.mdx:52]: We added this in the first place for a specific use that we don't believe is the best practice, so we don't want to guide users to use it anymore 🙂
It is still supported by the code, but we don't want it to be part of the documentation!
